src/components/Search/SearchChoice.tsx

```diff
@@ -37,15 +49,29 @@ const SearchChoice = () => {
             <Text>Save the recipes you like to your recipe gallery!</Text>
           </CardBody>
         </Card>
-        <Card cursor="pointer" onClick={navigateToOwnRecipe}>
+
+        <Card cursor="pointer" onClick={navigateToOwnRecipe} maxW="sm">
           <CardHeader>
-            <Heading size="md">ADD YOUR OWN RECIPE </Heading>
+            <Center>
+              <Heading size="md">ADD YOUR OWN RECIPE </Heading>
+            </Center>
           </CardHeader>
           <CardBody>
             <Text>Have a recipe you love?</Text>
             <Text>You can always type it out in our recipe friendly form.</Text>
           </CardBody>
         </Card>
+        {/* <Card cursor="pointer" onClick={navigateToOwnRecipe} maxW="sm">
+          <CardHeader>
+            <Center>
+              <Heading size="md">ADD BY URL </Heading>
+            </Center>
+          </CardHeader>
+          <CardBody>
+            <Text>Discovered a recipe you enjoy online? </Text>
+            <Text>Simply copy and paste the website URL right here.</Text>
+          </CardBody>
+        </Card> */}
```

Comment:
NON-BLOCKING: If this is a "for now" change, this is fine, but, if it's going to be a long time before this feature is added, I might just go ahead and delete it.